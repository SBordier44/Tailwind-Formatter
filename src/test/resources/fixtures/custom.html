<div class="last group-focus:fifth third second first sm:first fourth"></div>