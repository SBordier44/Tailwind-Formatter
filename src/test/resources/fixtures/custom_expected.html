<div class="first sm:first second third fourth group-focus:fifth last"></div>